games/devtest/mods/basetools/init.lua

```diff
@@ -420,10 +420,27 @@ minetest.register_tool("basetools:dagger_steel", {
 	}
 })
 
--- Test tool uses and punch_attack_uses
+-- Test tool uses, punch_attack_uses, and wear bar coloring
 local uses = { 1, 2, 3, 5, 10, 50, 100, 1000, 10000, 65535 }
+local wear_colors = { nil, nil, nil, "#5865f2", "slateblue", {blend=true, default="#ff00ff", [0.0]="red", [0.5]="yellow", [1.0]="blue"}, {
+                                                                                                                                               {
```

Comment:
Why so much indentation?

Reply:
Umm... not sure how that happened, will fix